src/Stratis.Bitcoin/P2P/PeerConnectorConnect.cs

```diff
@@ -60,18 +61,16 @@ public override void OnStartConnect()
         }
 
         /// <summary>
-        /// Only return nodes as specified in the -connect node arg.
+        /// Only connect to nodes as specified in the -connect node arg.
         /// </summary>
-        public override PeerAddress FindPeerToConnectTo()
+        public async override Task OnConnectAsync()
```

Comment:
elsewhere in the code we use `public override async` in that order, let's be consistent

Reply:
Done 👍